index.html

```diff
@@ -200,5 +200,33 @@ <h1>About</h1>
                         </div>
                     </div>
                 </section>
+
+                <section class="contactForm">
+                    <div class="contactLogo"></div>
+                    <div class="content">
+                        <h1>Get started</h1>
```

Comment:
- [OPTIONAL] I suggest you change this to any other heading `<hx> from <h2> to <h6>`. Your document should only have one `<h1>` element. Your browsers use this for seo optimization and also for your site to be used by screen readers. Read [here](https://ahrefs.com/blog/h1-tag/#:~:text=skim%20and%20digest.-,H1%20tags%20improve%20accessibility,title%20be%20an%20H1%20tag.) for more on this